frontend/src/metabase/entities/collections/collections.js

```diff
@@ -159,6 +159,9 @@ export function getExpandedCollectionsById(
   // "Our Analytics"
   collectionsById[ROOT_COLLECTION.id] = {
     ...ROOT_COLLECTION,
+    name: collectionsById[ROOT_COLLECTION.id]
+      ? ROOT_COLLECTION.name
+      : t`Collections`,
```

Comment:
I picked `Collections` since it what we use in the sidebar

Reply:
Nice choice and it's automatically be translated :)